packages/aws-cdk-lib/aws-logs/lib/log-group.ts

```diff
@@ -419,8 +419,9 @@ export interface LogGroupProps {
   /**
    * The class of the log group. Possible values are: STANDARD and INFREQUENT_ACCESS.
    *
-   * INFREQUENT_ACCESS class provides customers a cost-effective way to
-   * consolidate logs which supports querying using Logs Insights.
+   * INFREQUENT_ACCESS class provides customers a cost-effective way to consolidate 
```

Comment:
Hey @liangchenamz , thank you for opening a contribution with us. I see the build is failing due to a linting error. 

You can run `yarn lint --fix` in the aws-logs folder to fix this or just remove the extra space at the end of line 422.

Error: 
```
aws-cdk-lib: /codebuild/output/src1030512690/src/github.com/aws/aws-cdk/packages/aws-cdk-lib/aws-logs/lib/log-group.ts
aws-cdk-lib:   422:84  error  Trailing spaces not allowed  no-trailing-spaces
aws-cdk-lib: âœ– 1 problem (1 error, 0 warnings)
aws-cdk-lib:   1 error and 0 warnings potentially fixable with the `--fix` option.

Reply:
Done removing the extra space.